### DISCUSSION TITLE
feat: configurable buffer font settings

### DISCUSSION THREAD

**Discussion by taciturnaxolotl** [zed-industries/zed#7161] [Feature Requests]
It would be really cool to be able to theme comments differently from uncommented code and copilot suggestions differently from both of those; Adding additional font settings for each of those would be very helpful for fonts like GitHub Next's Monaspace font

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
